README.md

```diff
@@ -14,6 +14,37 @@ following actions are implemented
     $ git clone https://github.com/cognoma/figshare.git
     $ python setup.py install
 
+For development, I recommend:
+
+    $ python setup.py develop
+
+## Execution
+
+To use this Python module, here's a demonstration of initiating the API and
+retrieving information from Figshare API. This was done using
+[iPython](https://ipython.readthedocs.io/en/stable/), but
+other Python-based methods will work as well:
+
+    In [1]: from figshare.figshare import Figshare
+    In [2]: fs = Figshare()
+    In [3]: article_id = 4757131
+    In [4]: fs.get_article_details(article_id)
+
+That will produce a dictionary object.  Here's a snippet of the first and last few lines:
+
+    {'defined_type_name': 'dataset',
+     'embargo_date': None,
+     'citation': 'Lake, Sean; Wright, Edward L.; Assef, Roberto J.; Jarrett, Thomas H.; Petty, Sara; Stanford, Spencer A.; et al. (2019): Evolving Extragalactic Background and Luminosity Density. figshare. Dataset. https://doi.org/10.6084/m9.figshare.4757131.v1',
+     'url_private_api': 'https://api.figshare.com/v2/account/articles/4757131',
+...
```

Comment:
```suggestion
    ...
```

And delete the following blank line such that this code block renders properly in the GitHub markdown viewer.